instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/Instrumenter.java

```diff
@@ -58,7 +58,17 @@
       OpenTelemetry openTelemetry,
       String instrumentationName,
       SpanNameExtractor<? super REQUEST> spanNameExtractor) {
-    return new InstrumenterBuilder<>(openTelemetry, instrumentationName, spanNameExtractor);
+    return new InstrumenterBuilder<>(
+        openTelemetry, InstrumentationVersion.VERSION, instrumentationName, spanNameExtractor);
```

Comment:
I think you have switched name and version here

Reply:
@radcortez Please add new tests as well

Reply:
Ops.